internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -50,51 +45,59 @@ func New(z *zap.Logger, cfg *Config) (db.DB, error) {
 		return nil, err
 	}
 
-	db := &gormdb{z: z, cfg: cfg}
+	return &gormdb{z: z, cfg: cfg}, nil
+}
+
+func (db *gormdb) GormDB() (r, w *gorm.DB) { return db.rdb, db.wdb }
```

Comment:
Why named output variables?

Reply:
to make it clear which is writer and which is reader